### ISSUE TITLE
Allow multiple filters for subsetting

### ISSUE THREAD

**Issue by klkerr** [ABCD-STUDY/DEAP#71]
For instance, right now I can only select "Females only" or "Right-handed," but I can't select right-handed females without making a new filter in "Limit."

**Comment by HaukeBartsch** [comment]
Hi, yes. Your only option is to merge the two filters together. You can take the Females filter and the Right-handed filter and you would get a new filter (right handed females):
    (ehi_ss_score=1)and(sex="F")
You need to save that new filter. Hope that works.

**Comment by klkerr** [comment]
Yes, it'd just make things a lot easier when analyzing data if you could apply multiple filters at once within "Analyze," rather than having many different filters for all the different combinations.

**Comment by HaukeBartsch** [comment]
Yes, sounds like a good idea!
